src/Hyperlink/index.jsx

```diff
@@ -17,12 +17,13 @@ function Hyperlink(props) {
     externalLinkTitle,
     variant,
     isInline,
+    hideLaunchIcon,
     ...other
   } = props;
 
   let externalLinkIcon;
 
-  if (target === '_blank') {
+  if (target === '_blank' && !hideLaunchIcon) {
     // Add this rel attribute to prevent Reverse Tabnabbing
     other.rel = other.rel ? `noopener ${other.rel}` : 'noopener';
```

Comment:
these are important attribs that are needed don't skip them.

Reply:
fixed, moved condition inside if. for icon only.